src/sentry/snuba/occurrences_rpc.py

```diff
@@ -0,0 +1,52 @@
+import logging
+
+import sentry_sdk
+from sentry_protos.snuba.v1.request_common_pb2 import PageToken
+
+from sentry.search.eap.occurrences.definitions import OCCURRENCE_DEFINITIONS
+from sentry.search.eap.resolver import SearchResolver
+from sentry.search.eap.types import AdditionalQueries, EAPResponse, SearchResolverConfig
+from sentry.search.events.types import SAMPLING_MODES, SnubaParams
+from sentry.snuba import rpc_dataset_common
+
+logger = logging.getLogger("sentry.snuba.occurrences_rpc")
+
+
+class OccurrencesRPC(rpc_dataset_common.RPCBase):
+    DEFINITIONS = OCCURRENCE_DEFINITIONS
+
+    @classmethod
+    @sentry_sdk.trace
+    def run_table_query(
```

Comment:
If this is hitting the EndpointTraceItemTable and you're trying to get tags, it's incorrect.

You want the attribute names endpoint:

https://github.com/getsentry/sentry-protos/blob/main/proto/sentry_protos/snuba/v1/endpoint_trace_item_attributes.proto#L15-L41